app/scripts/modules/tencentcloud/src/pipeline/stages/disableCluster/disableClusterStage.ts

```diff
@@ -0,0 +1,11 @@
+import { validate } from './disableClusterValidators';
+import { Registry, IStageTypeConfig } from '@spinnaker/core';
+import { DisableClusterConfig } from './DisableClusterConfig';
+
+Registry.pipeline.registerStage({
+  provides: 'disableCluster',
+  key: 'disableCluster',
+  cloudProvider: 'tencentcloud',
+  component: DisableClusterConfig,
+  validateFn: validate,
+} as IStageTypeConfig);
```

Comment:
Why do we need this `as` assertion? Are we missing any expected properties? (Same question for other stage configs in this PR).

Reply:
I added it by mistake, now I have removed them at all